examples/seismic/acoustic/wavesolver.py

```diff
@@ -135,7 +128,7 @@ def adjoint(self, rec, srca=None, v=None, m=None, **kwargs):
         # Create the adjoint wavefield if not provided
         if v is None:
             v = TimeData(name='v', shape=self.model.shape_domain,
-                         save=False, time_order=self.time_order,
+                         save=False, time_order=2,
```

Comment:
I'm leaving the comment here, but this applies probably to other places as well. If you can, try to minimize the number of lines to instantiate a ``TimeData``. Sounds like stupid, but it makes the code incredibly verbose